### PR TITLE
[MIG] stock_picking_batch_extended: Migration scripts

### PR DESCRIPTION
**Pull request by pedrobaeza**
- Converts old stock.batch.picking in stock.picking.batch
- Restore link in stock.picking
- Other needed stuff

cc @Tecnativa TT18083

### CODE REVIEW

**Review by carlosdauden — APPROVED** [review]

File: stock_picking_batch_extended/migrations/12.0.1.0.0/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+from openupgradelib import openupgrade
+
+
+_field_renames = [
+    'stock.warehouse', 'stock_warehouse', 'default_picker_id',
```
Comment (sergio-teruel) [inline stock_picking_batch_extended/migrations/12.0.1.0.0/pre-migration.py:8]: Where is defined stock.wharehouse?

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

**Comment by sergio-teruel** [comment]
Fix travis
https://travis-ci.org/OCA/stock-logistics-workflow/jobs/623352690#L334

File: stock_picking_batch_extended/migrations/12.0.1.0.0/pre-migration.py (hunk 1 above)
Comment (pedrobaeza) [reply stock_picking_batch_extended/migrations/12.0.1.0.0/pre-migration.py:8]: I don't get you... `stock` module defines `stock.warehouse`, but this is only renaming a field..

**Comment by pedrobaeza** [comment]
Linter fixed

File: stock_picking_batch_extended/migrations/12.0.1.0.0/pre-migration.py (hunk 1 above)
Comment (sergio-teruel) [reply stock_picking_batch_extended/migrations/12.0.1.0.0/pre-migration.py:8]: Upps!! ok... I see.. model, table, old_field, renamed_field...
Sorry

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**